app/templates/index.html

```diff
@@ -90,6 +108,7 @@ <h2>Places we've been to</h2>
         </div>
     </div>
 
-    <script src="{{ url_for('static', filename='scripts/opentab.js') }}"></script> 
+    <script src="{{ url_for('static', filename='scripts/opentab.js') }}"></script>
+    <script src="{{ url_for('static', filename='scripts/navmenu.js') }}"></script> 
```

Comment:
By keeping these in the sections they're relevant to it might help with readability. Just keeps everything separated and to my knowledge has no impact on functionality.

Reply:
Oh, I thought it was generally recommended to place JavaScript code at the end of the `<body>` tag, because it allows the HTML content to load and render first before executing any JavaScript, what do you folks think?

Reply:
I believe the same functionality (render HTML before JS) can be achieved using "defer" attribute, while still allowing to place script anywhere on the HTML file. I think using "defer" would achieve both goals